src/Engine.cpp

```diff
@@ -1242,6 +1247,14 @@ void Engine::save_csv(
     }
 }
 
+void Engine::dump(
+        const QString& file_name,
+        bool clear)
+{
+    std::string file = utils::to_string(file_name).substr(7);   // file_name without file://
```

Comment:
Be careful with this. Check csv.cpp [40] to check how it should be done.
Also, there would be cool if such functionality is moved to a utils